### PR TITLE
Emptystringints

### PR DESCRIPTION
**Pull request by Balfa**
Internal story 307069

### CODE REVIEW

File: SqlReplay.Console/EventExecutor.cs [hunk 1]
```diff
@@ -394,12 +396,54 @@ private void SetupSqlCommandParameters(SqlCommand cmd, Rpc rpc)
...
+        /// '
+        /// DECLARE @alphaInt INT = 'abc'
+        /// DECLARE @alphaDt DATETIME = 'abc'
+        /// </code>
+        /// </remarks>
+        private static object SqlTypifyValue(object inputValue, SqlDbType sqlDbType)
+        {
+            return sqlDbType switch
+            {
+                SqlDbType.Int when inputValue is "" => 0,
```
Comment (benchamilton) [inline SqlReplay.Console/EventExecutor.cs:439]: In your example above you mention there can be any number of spaces in the input string and it translates to zero. Does this handle those cases or are you wanting to just limit to complete empty string here?

**Review by benchamilton — COMMENTED** [review]
Just the one question here

File: SqlReplay.Console/EventExecutor.cs (hunk 1 above)
Comment (Balfa) [reply SqlReplay.Console/EventExecutor.cs:439]: Bahh. My subconscious was expecting a question like that but I never bothered to proactively address it. At least we're totally on the same page, though... I really am just trying to laser focus this change on the empty string subset of the demonstrated SQL behaviors :)

**Comment by pareshdash** [comment]
Can you add me as a reviewer. it doesn't allow me to post a comment there

File: SqlReplay.Console/EventExecutor.cs (hunk 1 above)
Comment (pareshdash) [inline SqlReplay.Console/EventExecutor.cs:439]: do you need to trim blanks from inputvalue ?

Comment (Balfa) [reply SqlReplay.Console/EventExecutor.cs:439]: No, this is only intended to process empty string, by design.

**Review by pareshdash — APPROVED** [review]

**Pull request merged**